test/parallel/test-stdio-closed.js

```diff
@@ -10,20 +11,12 @@ if (common.isWindows) {
 
 if (process.argv[2] === 'child') {
   try {
-    process.stdout.write('stdout', function() {
-      try {
-        process.stderr.write('stderr', function() {
-          process.exit(42);
-        });
-      } catch (e) {
-        process.exit(84);
-      }
-    });
+    fs.fstatSync(1);
```

Comment:
should check `0` as well (the original didn't, possibly because its hard to do if input is unavailable).